app/src/main/java/org/mozilla/rocket/content/travel/data/YoutubeApiEntity.kt

```diff
@@ -0,0 +1,65 @@
+package org.mozilla.rocket.content.travel.data
+
+import org.json.JSONObject
+import org.mozilla.rocket.util.toJsonArray
+
+data class YoutubeApiEntity(val videos: List<YoutubeApiItem>) {
```

Comment:
The API result from back-end might have other video sources, so suggest to have a more general name like `VideoApiEntity`